index.es.md

```diff
@@ -0,0 +1,244 @@
+  <p><strong>Editor:</strong> <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Desarrollado por Grupo de Trabajo de Educación y Divulgación (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+# Read Translations Notes at https://github.com/w3c/wai-intro-accessibility/blob/gh-pages/README.md
+# end of translation instructions
+---
+
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
+{:/}
+
+Cuando los sitios y herramientas web están bien diseñados y codificados, las personas con discapacidad pueden utilizarlos. Sin embargo, en la actualidad muchos sitios y herramientas están desarrollados incluyendo barreras de accesibilidad que dificultan o imposibilitan su uso por parte de algunas personas.
+
+Hacer la web accesible beneficia tanto a las personas, como a las empresas y a la sociedad en general. Los estándares web internacionales definen lo que la accesibilidad necesita.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenido de la Página" class="full" %}
+{:/}
+
+<ul>
+<li><a href="#context">Accesibilidad en Contexto</a></li>
+<li><a href="#what">Qué es la Accesibilidad Web</a></li>
+<li><a href="#important">La Accesibilidad es Importante para las Personas, Empresas y Sociedad</a></li>
+<li><a href="#making">Hacer la Web Accesible</a></li>
+<li><a href="#evaluate">Evaluar Accesibilidad</a></li>
+<li><a href="#examples">Ejemplos</a></li>
+<li><a href="#more-info">Más Información</a></li>
+</ul>
+
+<span class="box-h box-h-simple box-h-full">Recursos relacionados</span><br>
+{% include video-link.html title="Video Introducción a la Accesibilidad Web y Estándares W3C <em>(4 minutos)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits.html" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Accesibilidad en Contexto {#context}
+
+<blockquote class="pull">
+  <p>El poder de la Web está en su universalidad.<br />
+    El acceso para cualquier persona, independientemente de las discapacidades, es un aspecto esencial.</p>
+  <footer><cite>Tim Berners-Lee, Director del W3C Director e inventor de la World Wide Web</cite></footer>
+</blockquote>
+
+La Web está fundamentalmente diseñada para trabajar para todo el mundo, independientemente del hardware, software, idioma, ubicación o capacidad. Cuando la Web cumple ese objetivo, es accesible para personas con un diverso rango de audición, movimiento, visión y habilidades cognitivas.
+
+Por lo tanto, el impacto de la discapacidad cambia radicalmente en la Web porque elimina las barreras de comunicación e interacción que muchas personas enfrentan en el mundo físico. Sin embargo, cuando los sitios web, aplicaciones, tecnologías o herramientas están mal diseñados, pueden crear barreras que excluyen a las personas del uso de la Web.
```

Comment:
Proposal:

"el impacto de la discapacidad cambia radicalmente en la Web porque **ésta** elimina las barreras de comunicación e interacción que muchas personas **encuentran** en el mundo físico"

Reply:
I agree with "encuentran" (much better than what I wrote initially), but I would avoid adding "ésta" since the subject for "elimina" is implied subject :/

Reply:
Yep, but it looks a bit ambiguous if the subject is left implied. The subject in the sentence is "el impacto de la discapacidad", so if it's not clear that in the subordinate clause the subject is "la Web", one might imply that the subject is "el impacto de la discapacidad". Don't you think so?

Notice how the original text in English makes clear what the subject is ("because the Web removes barriers […]").

Reply:
ok, you convinced me ;) and I prefer not to overthink of it :)
Thanks for the review Carlos